ibm/resource_ibm_pi_instance.go

```diff
@@ -60,6 +60,7 @@ func resourceIBMPIInstance() *schema.Resource {
 			},
 			"migratable": {
 				Type:        schema.TypeBool,
```

Comment:
I think as per desgin we added arguments starts with pi_...Should be add a new argument pi_migartable instead of migartable(may be we can mark this as derprecated)